### DISCUSSION TITLE
关于下一集的快捷键、重新播放

### DISCUSSION THREAD

**Discussion by sqq135** [hooke007/mpv_PlayKit#431] [mpv 通用]
因为看一部比较烧脑的剧，频繁去之前的剧集找线索，用这个 script-binding uosc/next 绑定作为下一集的快捷键，每次切换剧集都会从开始播，很麻烦。为排除其他干扰，仅保留load-plus脚本，也还是重新加载。没有在conf文件中找的相关参数，也不知道应该从哪里修改，请问应该如何处理？

或者换一个命令行？麻烦大佬了。

**Comment by hooke007** [comment]
https://mpv.io/manual/master/#command-interface-write-watch-later-config

**Reply by sqq135** [reply]
mpv.conf也有这个
 save-position-on-quit = yes
 watch-later-options = start,vid,aid,sid

是不是因为切换过快没有保存位置啊。但是时间长一点也还不行。

**Reply by hooke007** [reply]
我给你的是command，不是选项

**Comment by sqq135** [comment]
不好意思，我没有理解，换集还需要命令行吗？我想是不是可以在conf中修改？

write-watch-later这段是真没看懂，麻烦还需要再解释以下。

**Reply by hooke007** [reply]
command 就是写在 inputconf 里的命令

你这都把我问不明白了，你都知道  script-binding uosc/next  这个命令能用了，不知道别的命令怎么用吗

**Reply by sqq135** [reply]
是真不明白啊。现在知道了，之前都是遇到啥问题再查教程，没遇到的就不知道。那个教程虽然看完了一遍，但也半懂不懂。马上再学习下。

非常感谢

**Reply by hooke007** [reply]
官方的手册也不是断章取义看的，`write-watch-later-config` 位于 https://mpv.io/manual/master/#command-interface 类目下，而这个类目第一行就是讲写input.conf
